### PR TITLE
Modified to use angular-translate.

### PR DESCRIPTION
**Pull request by cynthialee**
@lkysow @amoghsa @Jareechang Feel free to merge if you get to this before I do.

### CODE REVIEW

File: src/angular/partials/search-results.html [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
 
-            <!-- hours section -->
-            <div class="col-xs-5">
-              <h3>Hours</h3>
-              <p> {{ getOpeningTime(result) || 'N/A'}} - {{ getClosingTime(result) || 'N/A' }} </p>
-            </div>
+                  <!-- hours section -->
+                  <div class="col-xs-5">
+                    <h3>{{ 'HOURS' | translate }}</h3>
+                    <p> {{ getOpeningTime(result) || 'N/A' }} - {{ getClosingTime(result) || 'N/A' }} </p>
```
Comment (cynthialee) [inline src/angular/partials/search-results.html:26]: Not sure how to nest the translation...

File: src/angular/partials/search-results.html [hunk 2]
```diff
@@ -1,41 +1,44 @@
 <!-- view when clicked once a category/region is click on -->
 
-<main role="main" data-ng-controller="ResultsCtrl">
```
Comment (lkysow) [inline src/angular/partials/search-results.html:3]: We don't need the `<main role="main">`?

**Comment by lkysow** [comment]
:+1: so how do the translators translate these strings now? Edit that config you were going to move to an external file?

File: src/angular/partials/search-results.html (hunk 2 above)
Comment (cynthialee) [inline src/angular/partials/search-results.html:3]: We already have it in index.html

**Comment by cynthialee** [comment]
Yes. There'd be another `$translateProvider.translations('ar')`. It'll be a copy-paste of the 'en_EN' version once we're ready to pass it to the translator, who would then replace all the right-hand strings.

**Comment by Jareechang** [comment]
nice!

**Pull request merged**